README.md

```diff
@@ -4,14 +4,20 @@
 [![YDB tests](https://github.com/ydb-platform/ydb-rs-sdk/actions/workflows/rust-tests.yml/badge.svg?branch=master&event=schedule)](https://github.com/ydb-platform/ydb-rs-sdk/actions/workflows/rust-tests.yml)
 
 Rust SDK for YDB.
-Supported rust: 1.60.0 and newer.
 
-Integration tests, with dependency from real YDB database marked as ignored.
-To run it:
-1. Set YDB_CONNECTION_STRING env
-2. run cargo test -- --include-ignored
+### Prerequisites
+Rust 1.60.0 or newer
+
+### Installation
+Add the YDB dependency to your project using `cargo add ydb` or add this your Cargo.toml:
+```toml
+[dependencies]
+ydb = "0.9.4"
```

Comment:
It needs write common version, for example "0.x.y" or update script for update versions while release:
https://github.com/ydb-platform/ydb-rs-sdk/blob/6bbdf8a3736126879be09720c1e746dc487df65a/.github/scripts/version-up.sh